linode/manager
# chore: Update `vite` and `sanitize-html` to update `postcss` for dependabot

## Description 📝

- Resolves https://github.com/linode/manager/security/dependabot/84

### Detailed Explanation
- Updates `vite` and `sanitize-html` so that  `postcss` consequently gets bumped to resolve https://github.com/linode/manager/security/dependabot/84. This works because `vite` and `sanitize-html` are the only two dependents on `postcss` and updating both of them will result in us getting the desired `postcss` version 


## Major Changes 🔄
- Updates `vite` to latest
- Updates `sanitize-html` (and its types) to latest
- Updates `@vitejs/plugin-react-swc` to latest

## How to test 🧪
- Verify that in the `yarn.lock`, postcss resolves to **only** version `8.4.31` or newer
- Verify automated checks pass
- Check out the PR and perform a quick check on Cloud Manager and Storybook
